### PR TITLE
Form create asso

### PR DESCRIPTION
**Pull request by LeoDiduch**

### CODE REVIEW

**Review by TheoBoschetti — DISMISSED** [review]
ok

**Review by BastienSaulnier — APPROVED** [review]
SnE

File: frontend/src/components/FormCreateCause.jsx [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    id_tag: []
+  });
+
+  const data = asso;
+
+  const handleSubmit = () => {
+    axios
+      .post(`${apiCall}/association`, data)
+      .then(res => {
+        alert(res.data);
```
Comment (LoicBrassart) [inline frontend/src/components/FormCreateCause.jsx:22]: Don't you think there are better ways to inform your visitors ? Like, a toast ?
(`react-toastify` is very easy to use and very nice to the eye !)

File: frontend/src/components/FormCreateCause.jsx [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+      })
+      .catch(err => {
+        alert(err);
```
Comment (LoicBrassart) [inline frontend/src/components/FormCreateCause.jsx:25]: Don't you think there are better ways to inform your visitors ? Like, a toast ?
(`react-toastify` is very easy to use and very nice to the eye !)

**Review by LoicBrassart — APPROVED** [review]
It works, but I really find the `alert` to be ugly, it'd be wise to change it !

**Pull request merged**